src/lib/ctable.lua

```diff
@@ -280,7 +282,7 @@ function CTable:add(key, value, updates_allowed)
    if self.occupancy + 1 > self.occupancy_hi then
       -- Note that resizing will invalidate all hash keys, so we need
       -- to hash the key after resizing.
-      self:resize(self.size * 2)
+      self:resize(max(self.size * 2, 10)) -- Could be current size is 0.
```

Comment:
When occupancy is higher than occupancy_hi size is at least 10 while if occupancy is lower than occupancy_lo size will be at least 1. Is that correct? Why 10?

Reply:
No reason.  We could start at 1 here if we want to.  Actually, I'll go ahead and do that; that way we get power-of-two sizes, which might be good for page alignment.